config/application.rb

```diff
@@ -40,22 +40,10 @@ class Application < Rails::Application
       html_tag
     }
 
-    # TODO: replace the origins wildcard to accept requets from find only?
-    # https://github.com/cyu/rack-cors#origin
-    config.middleware.insert_before 0, Rack::Cors do
-      allow do
-        origins '*'
-        resource '*', headers: :any, methods: [:get]
-      end
-    end
-
     config.autoload_paths += [Rails.root.join("app/workers")]
     config.autoload_paths += [Rails.root.join("lib/validators")]
     config.autoload_paths += [Rails.root.join("lib/ckan")]
 
     config.elasticsearch = config_for(:elasticsearch)
-
-    config.filter_parameters << :password
```

Comment:
These are duplicate in https://github.com/alphagov/datagovuk_publish/blob/master/config/initializers/filter_parameter_logging.rb too.

Are you removing them because we no longer have a form with `password` fields?

Reply:
We should remove this one as it's custom. The initializer comes from `rails g`, so I'd rather leave it.

Reply:
👍